Classes/MOC/Varnish/Service/ContentCacheFlusherService.php

```diff
@@ -30,11 +30,19 @@ class ContentCacheFlusherService {
 	protected $domainsToFlush = array();
 
 	/**
+	 * @Flow\InjectConfiguration("enabled")
+	 * @var boolean
+	 */
+	protected $enabled;
+
+	/**
 	 * @param NodeInterface $node The node which has changed in some way
 	 * @return void
 	 */
 	public function flushForNode(NodeInterface $node) {
-		$this->generateCacheTags($node);
+		if ($this->enabled) {
+			$this->generateCacheTags($node);
+		}
```

Comment:
Actually this should be moved to Package.php as I suggested in #29 

Please keep this PR open instead of opening a new one, just push a new commit on top of your branch and remember to pull first..